tests/frontend/test_frontend.py

```diff
@@ -0,0 +1,274 @@
+import os
+import pytest
+import subprocess
+
+GMT_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../..')
+
+from lib.global_config import GlobalConfig
+
+from tests import test_functions as Tests
+from playwright.sync_api import sync_playwright
+
+page = None
+context = None
+
+## Reset DB only once after module
+#pylint: disable=unused-argument
+@pytest.fixture(autouse=True, scope='module')
+def setup_and_cleanup_module():
+    # before
+
+    global page #pylint: disable=global-statement
+    global context #pylint: disable=global-statement
+    # start only one browser for whole file
+    playwright = sync_playwright().start()
+    browser = playwright.firefox.launch()
```

Comment:
firefox again

Reply:
thanks. description is wrong. Firefox is what I want